client/app/bundles/server-components/components/CommentsFeed.jsx

```
@@ -0,0 +1,99 @@
+
+  // Use lodash to process (stays on server)
+  const sortedComments = _.orderBy(comments, ['created_at'], ['desc']);
+  const recentComments = _.take(sortedComments, 10);
+
+  if (recentComments.length === 0) {
+    return (
+      <div className="bg-amber-50 border border-amber-200 rounded-lg p-6 text-center">
+        <p className="text-amber-700">
+          No comments yet. Add some comments from the{' '}
+          <a href="/" className="underline font-medium">home page</a> to see them rendered here
+          by server components.
```

ESLint

[error] Replace `home·page</a>·to·see·them·rendered·here⏎·········` with `⏎············home·page⏎··········</a>{'·'}⏎··········to·see·them·rendered·here`

(prettier/prettier)